x-pack/platform/plugins/shared/maps/public/_main.scss

```diff
@@ -1,26 +1,3 @@
-@import '../../../../../../src/core/public/mixins';
-
-// sass-lint:disable no-ids
-#maps-plugin {
-  @include kibanaFullBodyHeight();
-
-  display: flex;
-  flex-direction: column;
-  width: 100%;
-  overflow: hidden;
-}
-
-.mapFullScreen {
-  // sass-lint:disable no-important
-  height: 100vh !important;
-}
-
-#react-maps-root {
-  flex-grow: 1;
-  display: flex;
-  flex-direction: column;
-}
-
 .euiColorPicker__emptySwatch {
   position: relative;
 }
```

Comment:
I was not sure where this is leveraged, so I decided to keep it for now.

Reply:
That appears to be an EUI-specific css class; however, I find no reference of it the EUI repo. I tried jumping back in time to older versions, but its hard to search that way... still didn't find anything.

Given it is scoped to the Maps app (`#maps-plugin` id), and the color pickers on the layers seem to behave the same with it commented out (tested that), I personally feel confident in removing it.